modules/components/analytics/web/org.wso2.carbon.pc.analytics.web/src/web/js/actions.js

```diff
@@ -256,7 +307,7 @@ function drawExecutionTimeVsProcessInstanceIdResult(renderElement){
                         });
                     }
                     render(renderElementID, dataset, 'Execution Time (ms)', 'Process Instance Id');
-                }else{
+                } else {
                     //Error message
```

Comment:
If there's nothing to add in the else scope, better to remove the else section. No point having an empty scope.